app/models/alert.rb

```diff
@@ -3,7 +3,7 @@
 # Table name: alerts
 #
 #  id             :bigint           not null, primary key
-#  active         :boolean
+#  active         :boolean          default(FALSE), not null
```

Comment:
Could we add presence validation of the `active` attribute in the model and in spec?

Reply:
the field "activate" doesn't change in this PR. Maybe annotate was not run in the previous PR